cartography/intel/aws/waf.py

```diff
@@ -103,13 +148,27 @@ def cleanup_waf_classic_web_acls(neo4j_session: neo4j.Session, common_job_parame
 
 @timeit
 def sync_waf_classic(
-    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, current_aws_account_id: str,
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
     update_tag: int, common_job_parameters: Dict,
 ) -> None:
-    web_acls = get_waf_classic_web_acls(boto3_session)
-    transformed_acls = transform_waf_classic_web_acls(boto3_session, web_acls)
+    """
+    Syncs both regional and global WAF Classic ACLs.
+    """
+    all_web_acls = []
+
+    global_acls = get_waf_classic_global_web_acls(boto3_session)
+    all_web_acls.extend(global_acls)
+
+    for region in regions:
+        regional_session = boto3.Session(region_name=region)
```

Comment:
instead of using this way of instantiating a regional session, we should use the boto3_session object.

ex: 
```
client = boto3_session.client('sns', region_name=region, config=get_botocore_config())
```

See this in `sns.py` file line 59 / 77. We should follow the same way to instantiate sessions.